packages/element-lib/package.json

```diff
@@ -69,13 +69,12 @@
     "web3": "^1.2.2"
   },
   "devDependencies": {
+    "@transmute/eslint-config": "0.0.7",
     "dotenv": "^7.0.0",
-    "eslint": "^5.10.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jest": "^22.1.2",
-    "eslint-plugin-node": "^8.0.0",
-    "eslint-plugin-security": "^1.4.0",
+    "eslint": "^6.8.0",
+    "eslint-plugin-import": "^2.19.1",
+    "eslint-plugin-jest": "^23.2.0",
+    "eslint-plugin-node": "^11.0.0",
```

Comment:
Why are we removing `eslint-config-airbnb-base` and `eslint-plugin-security` ? they are both needed by eslint.config.node.json.

The eslint console in vscode can show what plugins are missing

Reply:
hmm, these should be added, I'm surprised to build passes without them.

Reply:
I don't see errors when editing element-lib locally or when running lint on the build server, this seems to imply that these are not required, but I added them anyway.